app/controllers/auth/SignedOutController.scala

```diff
@@ -20,15 +20,27 @@ import javax.inject.Inject
 import play.api.i18n.I18nSupport
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import views.html.{SignOutAnswersNotSavedView, SignOutView}
```

Comment:
Do it make sense for these to live in same directory as the original sign out screen -> views.html.auth._